client/src/i18n/en-us/index.js

```diff
@@ -207,4 +208,33 @@ export default {
   },
   general_failure: "Oops, there was an error.",
   loading: "Loading...",
+  /*********************
+   * Notifications follow the format of:
+   * <type>: {
+   *    short: "",
+   *    long: "",
```

Comment:
I notice there's no `long` properties included in the objects here. Is `long` still relevant?

Reply:
For now, yes. I anticipate different translation strings for the notifications page vs the pop up.